website/docs/hms.md

```diff
@@ -218,6 +228,38 @@ using query engines like `Presto` and/or `Trino`. Check out the guides for query
 SELECT * FROM iceberg_db.<table_name>;
 ```
 
+```sql md title="sql for Iceberg, Hudi and Delta Lake in S3"
+select * from hudi.hudi_db.people;
+select * from delta.delta_db.people;
+select * from iceberg.iceberg_db.people;
+```
+
+In the example docker image, you can find the catalog configurations in `/etc/trino/catalog`
+```shell md title="Trino Catalog with S3"
+trino@trino:/$ cat /etc/trino/catalog/hudi.properties 
```

Comment:
Can we add this here  https://github.com/apache/incubator-xtable/tree/main/demo/trino/catalog/hudi.properties ?

Reply:
Do we really need the cat shell command and this statement should suffice ? 
`In the example docker image, you can find the catalog configurations..`

Reply:
We can.  However the current demo with Trino doesn't have any S3 configs.  I have a open PR for a new demo with S3.  See https://github.com/apache/incubator-xtable/pull/459